_posts/2015-07-15-go-http.md

```diff
@@ -0,0 +1,549 @@
+---
+layout: post
+title: Go's http package by example
+tags: golang http middleware
+username: mediocregopher
+---
+
+# Go's http package by example
+
+Go's [http](http://golang.org/pkg/net/http/) package has turned into one of my
+favorite things about the Go programming language. Initially it appears to be
+somewhat complex, but in reality it breaks down into a couple simple components
```

Comment:
>  but in reality it breaks down into a couple simple components
Sounds weird to say "it breaks down". Maybe something like: "it can be broken down into..."?